creusot/tests/should_succeed/sum.rs

```diff
@@ -2,10 +2,10 @@ extern crate creusot_contracts;
 use creusot_contracts::*;
 
 #[requires(n@ < 1000)]
-#[ensures(result@ == n@ * (n@ + 1) / 2)]
+#[ensures(result@ * 2 == n@ * (n@ + 1))]
```

Comment:
```suggestion
#[ensures(result@ == n@ * (n@ + 1) / 2)]
```

I understand about the change bellow.  But this one should not be necessary.